mozetl/search/aggregates.py

```diff
@@ -191,6 +194,7 @@ def add_derived_columns(exploded_search_counts):
     addon_version:  The version of the followon-search@mozilla addon, or None
     '''
     udf_get_search_addon_version = udf(get_search_addon_version, StringType())
+    hll.register()
```

Comment:
Can we do this in `search_aggregates` near line 90 to avoid doing this for aggregates that don't have an HLL?